ocf_datapipes/utils/utils.py

```diff
@@ -35,6 +36,30 @@ def datetime64_to_float(datetimes: np.ndarray, dtype=np.float64) -> np.ndarray:
     return np.where(mask, nums, np.NaN)
 
 
+def datetime64_to_datetime(
```

Comment:
Like we talked about before, I would suggest using the xarray dates that already exist in the datasets directly, rather than converting and converting back. Its just less code to maintain, and might not be as brittle. So I would recommend removing this function, and using the built in dates instead.